### ISSUE TITLE
UnusedInheritedVariablePassedToClosure false-positive with string interpolation

### ISSUE THREAD

**Issue by EvgenyOrekhov** [slevomat/coding-standard#422]
```php
class C
{
    public function f($p)
    {
        return function () use ($p) {
            return "${p}";
        };
    }
}
```

Expected result: no warnings

Actual result:

> Unused inherited variable $p passed to closure.

Slevomat Coding Standard version: 4.7.0

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/fd3d8ffc42c97b5f7dac429c8e4406085288ec18
